src/dev-spaces-preview/azext_dev_spaces_preview/__init__.py

```diff
@@ -30,8 +30,8 @@ def load_arguments(self, _):
             c.argument('cluster_name', options_list=['--name', '-n'])
             c.argument('resource_group_name', options_list=['--resource-group', '-g'])
             c.argument('space_name', options_list=['--space', '-s'])
-            c.argument('parent_space_name', options_list=['--parent-space', '-p'])
             c.argument('update', options_list=['--update'], action='store_true')
+            c.argument('prompt', options_list=['--yes', '-y'], action='store_true')
```

Comment:
I'm still confused as to why we are registering the "az ads use" command. Doesn't this mean it will appear in the help? I thought we just internally call it from "az aks use-dev-spaces".